src/fbConfig.js

```diff
@@ -0,0 +1,12 @@
+const fbConfig = {
+  apiKey: "AIzaSyAU4xCVR44CNgsB47ljWvWb_yrmyW_VpAQ",
```

Comment:
You sure you want this checked in? Alternatively, you could set it as an environment variable http://stackoverflow.com/q/50047841

Reply:
I don't think there is any downside with this being publicly available: https://medium.com/@paulbreslin/is-it-safe-to-expose-your-firebase-api-key-to-the-public-7e5bd01e637b